jax/nn/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google LLC
+# Copyright 2019 Google LLC
```

Comment:
It doesn't really matter, so don't worry about changing it here, but I think we should leave copyright dates reflecting the time the file was first introduced rather than incrementing them.